Source/Hypermedia.MediaTypes/Hypermedia.MediaTypes.csproj

```diff
@@ -1,38 +0,0 @@
-<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
I hesitate removing the relations and media types. Siren media type is e.g. not in the default one from MS. Also I would like to consider completing the default relations from INNA specification
https://www.iana.org/assignments/link-relations/link-relations.xhtml

Reply:
This are constants like 'http' or 'https'. Yes - it is usefull to have them as a C# Constants. But I don't see the need for coupling packages. Me may copy them into the project. They never be changed.

Reply:
I'd suggest linking the two files into each project that needs them without creating separate packages OR we could create those packages as single import file packages (so that there is no *.dll but just a code file).

Reply:
Linking is a good idea

Reply:
I use them in product projects, so for me it is not purely internal.